packages/headless/src/features/facets/facet-search-set/facet-search-reducer-helpers.ts

```diff
@@ -18,6 +18,10 @@ export type FacetSearchState<T extends FacetSearchResponse> = {
   /** The initial maximum number of values to fetch.
    */
   initialNumberOfValues: number;
+  /**
+   * The unique identifier of the current request.
+   */
+  requestId?: string;
```

Comment:
Why optional ? I don't think it would be a breaking change ? 🤔 

edit: is it because you "reset" it to undefined on `handleFacetSearchClear` ?

Reply:
I'd prefer if the property was required too to communicate that the requestId property is always present.
We could set the requestId to an empty string when clearing,